app/test/std_sdk_test.go

```diff
@@ -0,0 +1,209 @@
+package app_test
+
+import (
+	"math/big"
+	"sync"
+	"testing"
+	"time"
+
+	"github.com/celestiaorg/celestia-app/app"
+	"github.com/celestiaorg/celestia-app/app/encoding"
+	"github.com/celestiaorg/celestia-app/testutil/testnode"
+	"github.com/cosmos/cosmos-sdk/crypto/hd"
+	"github.com/cosmos/cosmos-sdk/crypto/keyring"
+	"github.com/cosmos/cosmos-sdk/testutil/mock"
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	vestingtypes "github.com/cosmos/cosmos-sdk/x/auth/vesting/types"
+	banktypes "github.com/cosmos/cosmos-sdk/x/bank/types"
+	oldgov "github.com/cosmos/cosmos-sdk/x/gov/types/v1beta1"
+	stakingtypes "github.com/cosmos/cosmos-sdk/x/staking/types"
+	"github.com/ethereum/go-ethereum/common"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+	"github.com/stretchr/testify/suite"
+	abci "github.com/tendermint/tendermint/abci/types"
+)
+
+func TestStandardSDKIntegrationTestSuite(t *testing.T) {
+	if testing.Short() {
+		t.Skip("skipping test in unit-tests or race-detector mode.")
```

Comment:
[no change needed]

[`testing.Short()`](https://pkg.go.dev/testing#Short) will only return true when the `-short` flag is provided so this conditional won't be executed when `-race` is provided. However, I think this line was written with  awareness that https://github.com/celestiaorg/celestia-app/blob/ec5498c8cf3dc075b54182a986db72cdfd32eecb/Makefile#L89 only uses the `-race` flag when `-short` is also provided so this skip description seems fine. An alternative description may be:

```suggestion
		t.Skip("skipping SDK integration test in short mode.")
```